src/content/docs/fr/reference/image-service-reference.mdx

```diff
@@ -116,12 +116,13 @@ const service: LocalImageService = {
 			loading: attributes.loading ?? 'lazy',
 			decoding: attributes.decoding ?? 'async',
 		};
-	}
+	},
+  propertiesToHash: ['src', 'width', 'height', 'format', 'quality'], 
 };
 export default service;
 ```
 
-Au moment de la construction des sites statiques et des routes pré-rendues, `<Image />` et `getImage(options)` appellent la fonction `transform()`. Elles passent les options soit par les attributs du composant, soit par un argument `options`. Les images transformées seront compilées dans un dossier `dist/_astro`.
+Au moment de la construction des sites statiques et des routes pré-rendues, `<Image />` et `getImage(options)` appellent la fonction `transform()`. Elles passent les options soit par les attributs du composant, soit par un argument `options`. Les images transformées seront compilées dans un dossier `dist/_astro`.  Leurs noms de fichiers contiendront un hachage des propriétés passées à `propertiesToHash`. Cette propriété est optionnelle et sera par défaut `['src', 'width', 'height', 'format', 'quality']`. Si votre service d'image personnalisé a plus d'options qui modifient les images générées, ajoutez-les dans le tableau.
```

Comment:
nit: The beginning of the paragraph bothers me... (`<Image />` is a component (`un composant` in French) so it cannot be `Elles`).
```suggestion
Au moment de la construction des sites statiques et des routes pré-rendues, `<Image />` et `getImage(options)` appellent la fonction `transform()`. Ces derniers transmettent les options soit par les attributs du composant, soit par un argument `options`, respectivement. Les images transformées seront compilées dans un dossier `dist/_astro`. Leurs noms de fichiers contiendront un hachage des propriétés passées à `propertiesToHash`. Cette propriété est optionnelle et sera par défaut `['src', 'width', 'height', 'format', 'quality']`. Si votre service d'image personnalisé a plus d'options qui modifient les images générées, ajoutez-les dans le tableau.
```